src/store/balanceReducer.js

```diff
@@ -0,0 +1,17 @@
+const intialState = {
+    balance : 0
+}
+
+const balanceReducer = (state=intialState,action) => {
+switch(action.type){
+    case "DEPOSIT":
+        return {balance: state.balance + action.payload}
+    case "WITHDRAW":
+        return {balance: state.balance - action.payload}
+        case "LOADING":
+            return {...state, loading : true}    
```

Comment:
I don't see `loading` attribute in the initial state!! Do you need to do this in the other reducer ?